src/case-ui-toolkit.module.ts

```diff
@@ -76,13 +76,13 @@ import { CaseHistoryModule } from './shared/components/case-history';
       CaseViewComponent,
       CaseViewerComponent,
       CallbackErrorsComponent,
-      PaletteModule,
```

Comment:
Removing module from exports. It should not have been here. It is not related to this work, but been identified whilst here.